src/crypto/signature/envelope.rs

```diff
@@ -89,29 +89,6 @@ pub trait Envelope: Sized {
         Ok(w)
     }
 
-    /// Attempt to sign some payload with a given signer.
-    ///
-    /// # Arguments
-    ///
-    /// * `signer` - The signer to use to sign the payload.
-    /// * `payload` - The payload to sign.
-    ///
-    /// # Errors
-    ///
-    /// * [`SignError`] - the payload can't be encoded or the signature fails.
-    // FIXME ported
-    fn try_sign(
-        signer: &<Self::DID as Did>::Signer,
-        varsig_header: Self::VarsigHeader,
-        payload: Self::Payload,
-    ) -> Result<Self, SignError>
-    where
-        Ipld: Encode<Self::Encoder>,
-        Named<Ipld>: From<Self::Payload>,
-    {
-        Self::try_sign_generic(signer, varsig_header, payload)
-    }
-
```

Comment:
I like seeing red :smiling_imp: 
![image](https://github.com/fission-codes/rs-ucan/assets/1430958/66517ec8-f787-4b66-8e72-3dc867f375b5)

Reply:
haha yeah these used to behave differently but they became the same, so 💀